scripts/lib/CIME/namelist.py

```diff
@@ -2047,12 +2098,15 @@ def _parse_namelist_group(self):
             self._settings[group_name] = {}
         self._eat_whitespace()
         while self._curr() != '/':
-            name, values = self._parse_name_and_values()
+            name, values, addto = self._parse_name_and_values()
             dsettings = []
             if self._groupless:
                 if name in self._settings:
                     dsettings = self._settings[name]
-                values = merge_literal_lists(dsettings, values)
+                    if addto:
+                        values = self._settings[name] + values
+                if not addto:
```

Comment:
Sorry, I was thinking about _parse_namelist_group_name. I have now pushed that last change.